autoload/neomake/makers/ft/swift.vim

```diff
@@ -0,0 +1,11 @@
+function! neomake#makers#ft#swift#EnabledMakers()
+    return ['swiftlint']
+endfunction
+
+function! neomake#makers#ft#swift#swiftlint()
+    return {
+        \ 'args': ['lint', '--config', './.swiftlint.yml', '--quiet'],
```

Comment:
Is this a reasonable way to deal with the config file? Is there something better?